apps/full-site-editing/bin/sync-newspack-blocks.sh

```diff
@@ -124,9 +124,8 @@ cp -R $CODE/src/blocks/carousel $TARGET/blocks/
 cp -R $CODE/src/shared $TARGET/
 cp -R $CODE/src/components $TARGET/
 
-# Replace text domain
-find $TARGET/blocks/ \( -name \*.js -or -name \*.php \) -exec sed "${sedi[@]}" "s/, 'newspack-blocks' )/, 'full-site-editing' )/g" "{}" \;
-sed "${sedi[@]}" "s/'newspack-blocks',/'full-site-editing',/g" $TARGET/class-newspack-blocks.php
+# Fix the text domain
+npx eslint . --fix
```

Comment:
This should fix the JS text domain after a sync.

Reply:
will this work on files that are in the `.gitignore`?

Reply:
I assume so, AFAIK eslint is unaware of .gitignore